backends/core/workers.py

```diff
@@ -816,6 +828,109 @@ def _execute(self):
     job_name = '%s/jobs/%s' % (project_id, self._ml_job_id)
     self._enqueue('MLWaiter', {'job_name': job_name}, 60)
 
+class MLTrainer(MLWorker):
+  """Worker to train a ML model"""
+
+  PARAMS = [
+      ('project', 'string', True, '', 'ML project ID'),
+      ('jobDir', 'string', True, '',
+       'URI of folder to put output generated by AI platform (e.g. gs://bucket/folder)'),
+      ('packageUris', 'string', True, '', 
+        'URI of python package e.g. gs://bucket/folder/filename.tar.gz'),
+      ('trainFile', 'string', True, '',
```

Comment:
Please, replace all the parameters that go into `agrs` in the API call with a single multi-line parameter.

Reply:
I implemented this